### DISCUSSION TITLE
¿Qué forma recomiendan para tener los archivos en un repo?

### DISCUSSION THREAD

**Discussion by AlexAntonioG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#214] [Q&A] [answered]
Hola! , solo tenia la duda acerca de como están organizando sus trabajos, en mi caso pensaba hacer un repositorio de **FronEnd** en donde pondría una carpeta para cada semana, donde obviamente estarían los trabajos correspondientes a cada semana.

La duda surge porque veía que varias personas decían que harían un repositorio para cada semana, pero no se si eso se muy eficiente, cual creen que sea la mejor forma?

Gracias 😄

**Comment by AngelCruzO** [comment]
En mi caso estoy haciendo la opción de un solo repositorio y subiendo carpetas con cada tema, pensando en que si esto lo ve un reclutador, le sea más fácil visualizar la información; ya que si están en diferentes repositorios pueden desesperarse y no ver más el perfil de github.

**Reply by AlexAntonioG** [reply]
Si claro, igual pensaba en eso, siento que esa manera es más fácil y ordenada 🥇

**Comment by JonathanJRodriguez** [comment, marked as answer]
Yo estoy haciendo un repositorio por práctica y te explico porqué lo hice así, 
Al final del curso no recibiremos un certificado como ya lo han comentado, lo que obtendremos será este portafolio de habilidades que estamos dejando documentadas en github.
Entonces teniendo todo por separado es más fácil usar las evidencias con algún posible empleador o incluso si decidiera darle más detalle a alguna de las prácticas anteriores, por ejemplo, finalizar el sitio web de abogabot ya completamente funcional o hacer la página web de la pastelería con más detalles que iremos aprendiendo conforme avanza el curso, pues lo puedo hacer y queda como un proyecto independiente.
Entonces básicamente lo veo a futuro como un portafolio de proyectos y habilidades en lugar de verlo como la carpeta donde meto los trabajos del curso.
Saludos!

**Reply by AlexAntonioG** [reply]
No lo había pensado así, tienes mucha razón, gracias 😸
